src/requirements.yml

```diff
@@ -20,12 +20,18 @@
   name: nvme
 - src: https://github.com/cisagov/ansible-role-persist-journald
   name: persist_journald
+- src: https://github.com/cisagov/ansible-role-pip
+  name: pip
 - src: https://github.com/cisagov/ansible-role-python
   name: python
+  version: bugfix/install-python3-minimal
```

Comment:
You have a draft PR for this branch right now. Do you plan on switching to a regular PR and merging soon?

Reply:
The PR has been merged, so I have switched back to the default branch.